src/components/groups/groupsService.js

```diff
@@ -58,23 +61,12 @@
                     });
                 };
 
-                GroupsService.getAllGroups = (function () {
-                    var getAllGroupsProm =  _getStorage().get(_getGroupPath()).then(function (groups) {
-                        if(angular.equals(groups, {})){
-                            groups[GroupsService.defaultGroupName] = {
-                                name: GroupsService.defaultGroupName,
-                                groupKey: GroupsService.defaultGroupName
-                            };
-                        }
-                        GroupsService.groups = groups;
+                GroupsService.getAllGroups = function () {
+                    return storageSrv.get(GROUPS_PATH, groupsDefault()).then(function (groups) {
```

Comment:
the deleted code handle the default value for get.
In the new code the default value return from groupsDefault() (line 65)